crates/forge/src/lib.rs

```diff
@@ -310,7 +312,14 @@ fn run_tests_from_file(
     )
     .context("Failed setting up runner.")?;
 
-    pretty_printing::print_running_tests(&tests.relative_path, tests.test_cases.len());
+    let tests_source = if tests.relative_path == "src/lib.cairo" {
+        tests.package_name.as_str()
+    } else {
+        tests.relative_path.as_str()
+    };
```

Comment:
I would move this logic to `pretty_printing::print_running_tests` and just pass `tests` and `package_name` as args